.github/workflows/ci-cd.yml

```diff
@@ -166,8 +166,11 @@ jobs:
       matrix:
         os: [ubuntu, windows, macos]
         qemu: ['']
+        noextensions: ['0']
         include:
-          # Split ubuntu job for the sake of speed-up
+          # Add a pure python build and qemu builds
+        - os: ubuntu
+          noextensions: '1'
```

Comment:
This is not the right places to add it. It would essentially run the same wheel creation under each Python version + waste a CI job. It can be done together with the sdist. Although, that should also be updated as it still invokes `setup.py`.

I'm doing a round of maintenance updates across the repos, including packaging and CI changes. frozenlist and yarl are updated, multidict is next on my list. So I'll take care of this in a standardized way.